src/olympia/amo/admin.py

```diff
@@ -213,6 +213,9 @@ def changelist_view(self, request, extra_context=None):
                 'search_by_ip_actions_names': [
                     LOG_BY_ID[action].__name__ for action in self.search_by_ip_actions
                 ],
+                'minimum_search_terms_to_search_by_id': (
+                    self.minimum_search_terms_to_search_by_id
```

Comment:
well it's not technically a constant - it can be modified, though I've set it up to only be overwritten in the class definition of relevant `ModelAdmin`s... I've tried to define it close or other search-related properties.